public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/route-settings/RouteSettings.tsx

```diff
@@ -131,20 +129,13 @@ export const RoutingSettings = ({ alertManager }: RoutingSettingsProps) => {
   );
 };
 
-function useGetDefaultsForRoutingSettings() {
```

Comment:
There is no need (and we want to avoid this in the alert rule form) to fetch this from the alert manager configuration.
For simplified routing groupBy will be always the same by default.
There is another [PR](https://github.com/grafana/grafana/pull/82050) taking care of this.